modules/ROOT/pages/install-acm.adoc

```diff
@@ -4,9 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-NOTE: Installing, updating or configuring API Community Manager must be done by a user account with the *System Administrator* profile in the ACM Salesforce organization.
+NOTE: Installing, updating or configuring API Community Manager (ACM) must be done by a user account with the *System Administrator* profile in the ACM Salesforce organization.
```

Comment:
Intentionally passive?
Also, consider just merging this requirement into the task intro below.

Reply:
Also, use the Oxford comma: Installing, updating, or configuring